### PR TITLE
Refs #54585 admin create new user

### PR DESCRIPTION
**Pull request by ggbond2605**

### CODE REVIEW

**Comment by NeverSmileK57CLC** [comment]
OK

File: config/routes.rb [hunk 1]
```diff
@@ -2,4 +2,9 @@
   devise_for :users, controllers: {registrations: :registrations}
   root "static_pages#home"
   get "help" => "static_pages#help"
+
+  namespace :admin do
+    root "static_pages#home"
```
Comment (hellovietnam93) [inline config/routes.rb]: users#index

File: app/views/admin/users/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :password_confirmation, t(:confirm) %>
+      <%= f.password_field :password_confirmation, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :role %>
+      <%= f.select(:role, Settings.role) %>
```
Comment (hellovietnam93) [inline app/views/admin/users/_form.html.erb]: remove ()

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**